cmd/snap-bootstrap/export_test.go

```diff
@@ -55,10 +56,10 @@ func MockOsutilIsMounted(f func(path string) (bool, error)) (restore func()) {
 }
 
 func MockRunMnt(newRunMnt string) (restore func()) {
-	oldRunMnt := runMnt
```

Comment:
perhaps not for this PR, but do we still need MockRunMnt here given that dirs.RunMnt is now underneath `s.rootdir` in most tests?